### DISCUSSION TITLE
qBitrr grabs new torrent within minutes.

### DISCUSSION THREAD

**Discussion by devanteweary** [Feramance/qBitrr#143] [Q&A] [answered]
I noticed sometimes a torrent will start out stalled but will stabalized within 10 minutes or so. Maybe even after 30 minutes.

Well I see that qBitrr will see this stalled state and grab a new torrent right away - within minutes - of the original torrent being added.  
When if it would have waited, it wouldn't have needed to grab a new one.

This ends up with multiple torrents for the same media being downloaded twice usually.

Is there a way to change this behavior? I kinda thought that's what the IgnoreifYounger option did, but qBitrr is either ignoring that option or I'm not using it correctly (related to when I was asking how that option worked a few weeks ago).

See below for an example. 
Notice the bottom torrent is the first one and the date/times are when they were added.  
And the second one was added a mere 4 minutes later. 

So I'm sure what happened is the first one started off stalled, then qBitrr added a second one, then the first one stabilized quickly.

![image](https://github.com/user-attachments/assets/a052ebbd-694e-4753-be0d-6f12bbbcf26e)

And here's my config.

<details>
<summary>Config</summary>

[Settings]
ConsoleLevel = "INFO"
Logging = true
CompletedDownloadFolder = "/downloads"
FreeSpace = "-1"
NoInternetSleepTimer = 600
LoopSleepTimer = 303
SearchLoopDelay = 43200
AutoPauseResume = false
FailedCategory = "failed"
RecheckCategory = "recheck"
Tagless = true
IgnoreTorrentsYoungerThan = 3600
PingURLS = ["one.one.one.one", "dns.google.com"]
FFprobeAutoUpdate = true

[qBit]
Disabled = false
Host = "192.168.1.200"
Port = 8080
UserName = "devante"

[Sonarr-TV]
Managed = true
URI = "http://192.168.1.200:8989"
APIKey = "XXXXXX"
Category = "Videos - Series"
ReSearch = true
importMode = "Copy"
RssSyncTimer = 0
RefreshDownloadsTimer = 0
ArrErrorCodesToBlocklist = []

[Sonarr-TV.EntrySearch]
SearchMissing = true
AlsoSearchSpecials = true
SearchLimit = 3
SearchByYear = true
SearchInReverse = false
SearchRequestsEvery = 3599
DoUpgradeSearch = false
QualityUnmetSearch = false
CustomFormatUnmetSearch = false
ForceMinimumCustomFormat = false
SearchAgainOnSearchCompletion = true
UseTempForMissing = false
MainQualityProfile = "High Definition"
SearchBySeries = false
PrioritizeTodaysReleases = true

[Sonarr-TV.EntrySearch.Overseerr]
SearchOverseerrRequests = false
OverseerrURI = "http://192.168.1.200:5055"
OverseerrAPIKey = "XXXXXX=="
ApprovedOnly = true
Is4K = false

[Sonarr-TV.Torrent]
CaseSensitiveMatches = false
FolderExclusionRegex = []
FileNameExclusionRegex = []
FileExtensionAllowlist = []
AutoDelete = false
IgnoreTorrentsYoungerThan = 3600
MaximumETA = -1
MaximumDeletablePercentage = 0.75
DoNotRemoveSlow = true

AllowStalled = true
ReSearchStalled = true
StalledDelay = 0
Trackers = []

[Sonarr-TV.Torrent.SeedingMode]
DownloadRateLimitPerTorrent = -1
UploadRateLimitPerTorrent = -1
MaxUploadRatio = -1
MaxSeedingTime = -1
RemoveTorrent = -1
RemoveDeadTrackers = false
RemoveTrackerWithMessage = ["skipping tracker announce (unreachable)", "No such host is known", "unsupported URL protocol", "info hash is not authorized with this tracker"]

[[Sonarr-Anime.Torrent.Trackers]]
Name = "Nyaa"
Priority = 10
URI = "http://nyaa.tracker.wf:7777/announce"
MaximumETA = 18000
DownloadRateLimit = -1
UploadRateLimit = -1
MaxUploadRatio = -1
MaxSeedingTime = -1
AddTrackerIfMissing = false
RemoveIfExists = false
SuperSeedMode = false
AddTags = ["qBitrr-anime"]

[Radarr-1080]
Managed = true
URI = "http://192.168.1.200:7878"
APIKey = "XXXXXXX"
Category = "Videos - Movies"
ReSearch = true
importMode = "Auto"
RssSyncTimer = 0
RefreshDownloadsTimer = 0
ArrErrorCodesToBlocklist = []

[Radarr-1080.EntrySearch]
SearchMissing = true
AlsoSearchSpecials = true
SearchLimit = 5
SearchByYear = true
SearchInReverse = false
SearchRequestsEvery = 3599
DoUpgradeSearch = false
QualityUnmetSearch = false
CustomFormatUnmetSearch = false
ForceMinimumCustomFormat = false
SearchAgainOnSearchCompletion = true
UseTempForMissing = false
MainQualityProfile = "High Definition"

[Radarr-1080.EntrySearch.Overseerr]
SearchOverseerrRequests = false
OverseerrURI = "http://192.168.1.200:5055"
OverseerrAPIKey = "XXXXXXX=="
ApprovedOnly = true
Is4K = false

[Radarr-1080.Torrent]
CaseSensitiveMatches = false
FolderExclusionRegex = []
FileNameExclusionRegex = []
FileExtensionAllowlist = []
AutoDelete = false
IgnoreTorrentsYoungerThan = 3600
MaximumETA = -1
MaximumDeletablePercentage = 0.75
DoNotRemoveSlow = true

AllowStalled = true
ReSearchStalled = true
StalledDelay = 0

[Radarr-1080.Torrent.SeedingMode]
DownloadRateLimitPerTorrent = -1
UploadRateLimitPerTorrent = -1
MaxUploadRatio = -1
MaxSeedingTime = -1
RemoveTorrent = -1
RemoveDeadTrackers = false
RemoveTrackerWithMessage = ["skipping tracker announce (unreachable)", "No such host is known", "unsupported URL protocol", "info hash is not authorized with this tracker"]

[[Radarr-1080.Torrent.Trackers]]
Name = "Rarbg-2810"
Priority = 1
URI = "udp://9.rarbg.com:2810/announce"
MaximumETA = 18000
DownloadRateLimit = -1
UploadRateLimit = -1
MaxUploadRatio = -1
MaxSeedingTime = -1
AddTrackerIfMissing = false
RemoveIfExists = false
SuperSeedMode = false
AddTags = ["qBitrr-Rarbg", "Movies and TV"]

</details>

Thanks for any advice!

**Comment by Feramance** [comment]
I'll check and confirm that the stalled logic obeys the `IgnoreTorrentsYoungerThan` config

**Reply by Feramance** [reply, marked as answer]
FYI, this should be resolved

**Reply by devanteweary** [reply]
> FYI, this should be resolved

Oh nice man. I had kinda just ignored it since it wasn't a big problem. 
Thanks for taking care of it. :>
